_data/tournaments.tsv

```diff
@@ -4,6 +4,7 @@ start-date	end-date	variant	location	tournament	source
 2024-02-24	2024-02-24	Shogi	Berlin-Mitte, Germany	9th Mori Ogai Tournament Berlin	<a href='http://fesashogi.eu/index.php?mid=2'>fesashogi.eu</a>
 2024-04-13	2024-04-13	Shogi	BoardGames Ruse, Ruse, Bulgaria	Ruse Shogi Open 2024	<a href='http://fesashogi.eu/index.php?mid=2'>fesashogi.eu</a>
 2024-04-27	2024-04-27	Shogi	Essen, Nordrhein-Westfalen, Germany	EMO Shogi Open 2024	<a href='http://fesashogi.eu/index.php?mid=2'>fesashogi.eu</a>
+2024-05-04  2024-05-05  Shogi Milan, Italy 20th Italian Shogi Championship <a href='https://www.associazioneitalianashogi.it/2024/01/17/xx-campionato-italiano-di-shogi-xx-italian-shogi-championship-4-5-may-2024/'>associazioneitalianashogi.it</a>
```

Comment:
Thanks. It looks like the line uses whitespaces instead of a tabs though. Can you please check?

Reply:
But other than that feel free to commit such changes directly, you should have the permissions I think.

Reply:
Oh, I completely forgot that :)

Reply:
This reminded me to add some linting of the YAML and TSV data since they are somewhat susceptible to whitespace typos. Hopefully this should automatically point out such errors in the future.